src/overlays/kaleido_scope/ovl_kaleido_scope/z_kaleido_scope_NES.c

```diff
@@ -502,8 +516,964 @@ s32 D_8082BEB8[] = { 0, 0 };
+                        pauseCtx->state = PAUSE_STATE_MAIN;
+                        pauseCtx->savePromptState = PAUSE_SAVEPROMPT_STATE_0;
+                        pauseCtx->roll = -314.0f;
+                        pauseCtx->itemPageRoll = pauseCtx->mapPageRoll = pauseCtx->questPageRoll =
+                            pauseCtx->maskPageRoll = 0.0f;
+                        func_8011552C(play, 0x15);
+                        gSaveContext.buttonStatus[EQUIP_SLOT_B] = D_801C6A98[pauseCtx->pageIndex + 1][0];
+                        gSaveContext.buttonStatus[EQUIP_SLOT_C_LEFT] = D_801C6A98[pauseCtx->pageIndex + 1][1];
+                        gSaveContext.buttonStatus[EQUIP_SLOT_C_DOWN] = D_801C6A98[pauseCtx->pageIndex + 1][1];
+                        gSaveContext.buttonStatus[EQUIP_SLOT_C_RIGHT] = D_801C6A98[pauseCtx->pageIndex + 1][1];
+                        if ((pauseCtx->cursorSpecialPos == PAUSE_CURSOR_PAGE_LEFT) ||
+                            (pauseCtx->cursorSpecialPos == PAUSE_CURSOR_PAGE_RIGHT)) {
+                            KaleidoScope_MoveCursorToSpecialPos(play, pauseCtx->cursorSpecialPos);
+                        } else {
+                            func_80821A04(play);
+                        }
+                    }
+                    break;
+
+                case PAUSE_SAVEPROMPT_STATE_6:
+                    if (interfaceCtx->unk_264 != 255) {
+                        interfaceCtx->unk_264 += 10;
+                        if (interfaceCtx->unk_264 >= 255) {
+                            interfaceCtx->unk_264 = 255;
+                            pauseCtx->state = PAUSE_STATE_OFF;
+                            Game_SetFramerateDivisor(&play->state, 3);
+                            R_PAUSE_BG_PRERENDER_STATE = PAUSE_BG_PRERENDER_MAX;
+                            Object_LoadAll(&play->objectCtx);
+                            BgCheck_InitCollisionHeaders(&play->colCtx, play);
+                            STOP_GAMESTATE(&play->state);
+                            SET_NEXT_GAMESTATE(&play->state, TitleSetup_Init, sizeof(TitleSetupState));
+                            func_801A4058(0x14);
+                            gSaveContext.seqId = (u8)NA_BGM_DISABLED;
+                            gSaveContext.ambienceId = AMBIENCE_ID_DISABLED;
+                        }
+                    }
+                    break;
+            }
+            break;
+
+        case PAUSE_STATE_GAMEOVER_2:
+            pauseCtx->cursorSlot[PAUSE_MAP] = R_REVERSE_FLOOR_INDEX + DUNGEON_FLOOR_INDEX_4;
+            pauseCtx->cursorPoint[PAUSE_MAP] = R_REVERSE_FLOOR_INDEX + DUNGEON_FLOOR_INDEX_4;
+            sPauseCursorLeftX = -175;
+            sPauseCursorRightX = 155;
+            pauseCtx->roll = -434.0f;
+            Interface_SetHudVisibility(HUD_VISIBILITY_NONE);
+
+            pauseCtx->iconItemSegment =
+                (void*)(((uintptr_t)play->objectCtx.spaceStart + 0x30) & ~0x3F); // Messed up ALIGN64
```

Comment:
Do we need a bug tag for this?

Reply:
I'm not sure? There are many instances of these "aligns" that are off just like this, and some cause issues and others don't, but I don't know enough if this actually causes issues or is just a quirk

Reply:
Just label them for now and we can tackle them as a group later.

Reply:
Agreed, I just say put a bug tag for now and we can figure it out later.

Reply:
No this isn't a bug thanks to `spaceStart` being 16-aligned (at least in OoT `GameState_Alloc` 16-aligns allocs)

My theory is whoever wrote this thought it may be wasteful to do +0x3F &~0x3F when the address being aligned already is aligned to a lower alignment (spoiler: it isn't wasteful)

Reply:
Interesting, good to know. Thanks for clarifying 🙂